vnda/loaders/productList.ts

```diff
@@ -19,6 +19,9 @@ export interface Props {
   /** @description search for products that have certain tag */
   tags?: string[];
 
+  /** @description search for products that have certain type_tag */
+  type_tags?: { key?: string; value?: string }[] | undefined;
```

Comment:
Padrao de nomeação de variaveis é camelCase.
```suggestion
  /** @description search for products that have certain type_tag */
  typeTags?: { key?: string; value?: string }[] | undefined;
```